README.md

```diff
@@ -55,6 +55,11 @@ created Zarr store.*
 
 1. *`HARMONY_HOST_URL` is the harmony base url for your target environment. e.g. `SIT` would be `https://harmony.sit.earthdata.nasa.gov`*
 
+1. When testing against a local Harmony installation, where
+  `HARMONY_HOST_URL=http://localhost:3000`, best testing results occur when
+  running the notebook manually on a Jupyter notebook server, rather than via
+  the `run_notebooks.sh` script.
```

Comment:
I'm pretty sure you can't run the run_notebooks.sh script against local.  docker in docker problem?

Reply:
Okay - I can make this statement more accurate (e.g., "you can't" rather than "it's not as good").

Reply:
[Updated here](https://github.com/nasa/harmony-regression-tests/pull/79/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R58-R61)